codegen/method.go

```diff
@@ -198,12 +198,15 @@ func (ms *MethodSpec) setRequestType(curThriftFile string, funcSpec *compile.Fun
 		ms.RequestType, err = packageHelper.TypeFullName(
 			curThriftFile, funcSpec.ArgsSpec[0].Type,
 		)
+		if err == nil && isStructType(funcSpec.ArgsSpec[0].Type) {
```

Comment:
typedef should be covered since the type assertion is against the `RootTypeSpec` of given spec. https://github.com/uber/zanzibar/blob/master/codegen/method.go#L217